src/kOS/Function/Misc.cs

```diff
@@ -28,6 +28,51 @@ public override void Execute(SharedObjects shared)
         }
     }
 
+[Function("hudtxt")]
```

Comment:
I prefer hudtext. I changed it early on when I ran into a naming conflict in development. It just stuck.